Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -7,16 +7,37 @@ namespace BootCamp.Chapter
     /// <summary>
     /// Caesar cipher which supports all ASCII character (256 in total)
     /// </summary>
-    public static class CaesarCipher
+    public class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return EncryptString(message, shift);
         }
 
+        private static string EncryptString(string message, int shift)
+        {
+            if (message == null)
+            {
+                return null;
+            }
+
+            List<char> encryptString = new List<char>();
+
+            for (int j = 0; j < message.Length; j++)
+            {
+                var messageToInt = (int)message[j];
+                var messageToIntAddShift = (messageToInt + shift);
+                var messageEncrypt = (char)messageToIntAddShift;
+                encryptString.Add(messageEncrypt);
+                Console.WriteLine(messageEncrypt);
+            }
+
+            return new string(encryptString.ToArray());
+
+        }
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            return EncryptString(message, 0 - shift);
```

Comment:
The fact that you spotted that you can reuse EncryptString puts you ahead of many. Well done.